hazelcast/src/main/java/com/hazelcast/internal/util/phonehome/PhoneHome.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hazelcast.internal.util.phonehome;
+
+import com.hazelcast.instance.impl.Node;
+import com.hazelcast.logging.ILogger;
+import com.hazelcast.spi.properties.ClusterProperty;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.RejectedExecutionException;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import static java.lang.System.getenv;
+
+/**
+ * Pings phone home server with cluster info daily.
+ */
+public class PhoneHome {
+
+    private static final String FALSE = "false";
+    private static final String DEFAULT_BASE_PHONE_HOME_URL = "http://phonehome.hazelcast.com/ping";
```

Comment:
Can we switch to HTTPS please?

Reply:
Do we have https for the service? cc @erosb

Reply:
AFAIK we currently don't have it. We can handle it separately. We can force redirect to https once that's done for older IMDG versions.

Reply:
I would like to see it resolved before merging this PR. We shouldn't leak even more info about users environment.
We can start with both endpoints available (http, https) - i.e. no redirect is needed, but new Hazelcast releases should go to the encrypted one.

Reply:
I created a callhome issue to track this: https://github.com/hazelcast/callhome/issues/35